.gitignore

```diff
@@ -0,0 +1,9 @@
+*.class
+Assignment2\
```

Comment:
you can merge your previous assignment into master or main branch and can create new branch for new assignment from master or main branch

Reply:
> ok sir i will keep it in mind for next assignments

Reply:
> you can merge your previous assignment into master or main branch and can create new branch for new assignment from master or main branch

yes sir I'll keep this in mind for further assignments.